ddl/partition.go

```diff
@@ -217,23 +224,34 @@ func checkCreatePartitionValue(ctx sessionctx.Context, tblInfo *model.TableInfo,
 
 // getRangeValue gets an integer from the range value string.
 // The returned boolean value indicates whether the input string is a constant expression.
-func getRangeValue(ctx sessionctx.Context, tblInfo *model.TableInfo, str string) (int64, bool, error) {
-
-	if value, err := strconv.ParseInt(str, 10, 64); err == nil {
-		return value, false, nil
-	}
+func getRangeValue(ctx sessionctx.Context, tblInfo *model.TableInfo, str string, unsignedBigint bool) (interface{}, bool, error) {
```

Comment:
how about return uint64 not interface{}?

Reply:
@crazycs520 Because it might be `int64` or `uint64`.

Reply:
int64 can use  `uint64( int64Variable_name )` conver to uint64.

Reply:
@crazycs520 Conversion from signed `int64` to `uint64` can be an error.